activiti-cloud-notifications-graphql/templates/_helpers.tpl

```diff
@@ -27,6 +27,21 @@ Create a default service name.
 {{- printf "%s" $name | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
+{{/*
```

Comment:
@igdianov why do we need this? even if the answer is .."because we have it in all the other templates" I need to understand what are the implications for having an ingress path for the actuator.

Reply:
I asked for it as we check it for service version etc besides the consistency with others

it's more complicated than it should be in helm as the notification service is mounted at the app root, another option would be to just have a single path at /notifications and anything underneath would be published, @igdianov is this a viable option?

Reply:
@mteodori The main `ingress.path` value allows to set the base path for /graphql, /ws/graphql, /graphiql and /actuator endpoints. You can choose to template `ingress.path` together with `SERVER_SERVLET_CONTEXTPATH` environment variable to expose Notification service on any path if needed, i.e.

To expose `/notifications` base path, the following configuration should be applied:

```yaml
ingress:
  path: /notifications
extraEnv: |
  - name: SERVER_SERVLET_CONTEXTPATH
     value: {{ include "common.ingress-path" . }}
```

We can also override `ingress.path` with `/{{ .Release.Name }}/notifications` template to set scope to support AAE deployments.

Reply:
I also think it would make sense to provide `SERVER_SERVLET_CONTEXTPATH` env as part of the deployment spec to ensure it is not overridden by mistake with `extraEnv`.